ui/src/components/page.gts

```diff
@@ -54,9 +54,7 @@ export class Page extends Component<{
   <template>
     {{#if this.selected.hasError}}
       {{yield this.selected.error to='error'}}
-    {{/if}}
```

Comment:
these do need to be separate, as you can have prose rendered _and_ an error present as well (prose of previous page, error of next)